index.html

```diff
@@ -18,7 +18,19 @@
 
 </head>
 <body>
-<div id="ucsf-banner-nav" class="logo">
+<div id="top-header-region" class="logo">
```

Comment:
this is making it worse. if anything, we should revert to a top-level class name that can be assumed unique in any context and carries some valuable semantic information. 
proposing instead 

```css
<div class="ucsf-banner-nav logo">
```